index.html

```diff
@@ -2277,42 +2277,42 @@ <h5>
               <p data-lang="zh-hant">文字橫排、注音置於基字右側時，段落間的漢字皆須預留基字尺寸一半以上的字距，每字皆然，注文應標注於此字距間。在縱橫對齊的原則下，未標注注文的文字（如標點符號、西文詞組、首行縮進的字數等）也會預留相同的空間。</p>
             </div>
           </li>
           <li>
-            <p data-lang="en">In principle, the size of Zhuyin annotations, including the phonetic symbols and the tone marks, should be 25% of the size of the base characters. In vertical Zhuyin, Zhuyin annotations should be vertically center-aligned to the base character, while in horizontal Zhuyin, Zhuyin annotations should be center-aligned over the base character beneath them.</p>
-            <p data-lang="zh-hans" class="checkme">原则上，注文中各个符号（注音及调号）的尺寸为基字的25%。直排注音时，注文整体垂直向基字置中对齐；横排注音时，注文整体水平向基字居中对齐。</p>
-            <p data-lang="zh-hant">原則上，注文中各個符號（注音及調號）的尺寸為基字的25%。直排注音時，注文整體垂直向基字置中對齊；橫排注音時，注文整體水平向基字置中對齊。</p>
+            <p data-lang="en">In principle, the size of Zhuyin annotations (including the phonetic symbols and the tone marks) to that of base characters ratio should be 3:10. In vertical Zhuyin, Zhuyin annotations should be vertically center-aligned to the base character, while in horizontal Zhuyin, Zhuyin annotations should be center-aligned over the base character beneath them.</p>
+            <p data-lang="zh-hans">原则上，注文中各个符号（注音及调号）的尺寸与基字比3:10。直排注音时，注文整体垂直向基字置中对齐；横排注音时，注文整体水平向基字居中对齐。</p>
+            <p data-lang="zh-hant">原則上，注文中各個符號（注音及調號）的尺寸與基字比3:10。直排注音時，注文整體垂直向基字置中對齊；橫排注音時，注文整體水平向基字置中對齊。</p>
             <div class="note">
               <p data-lang="en">The percentage rules above assume that all the tone marks (ˊˇˋ˪˫ㆴㆵㆶㆷ) and the neutral tone mark (˙) are set with the same font size as that of the base characters.</p>
-              <p data-lang="zh-hans" class="checkme">所用字体中，平上去入声（ˊˇˋ˪˫ㆴㆵㆶㆷ）及轻声（ ̇）等调号皆应与汉字尺寸一致，方适用以上比例。</p>
+              <p data-lang="zh-hans">所用字体中，平上去入声（ˊˇˋ˪˫ㆴㆵㆶㆷ）及轻声（˙）等调号皆应与汉字尺寸一致，方适用以上比例。</p>
               <p data-lang="zh-hant">所用字體中，平上去入聲（ˊˇˋ˪˫ㆴㆵㆶㆷ）及輕聲（˙）等調號皆應與漢字尺寸一致，方適用以上比例。</p>
             </div>
             <div class="note">
-              <p data-lang="en">In vertical Zhuyin, the width of the space taken by the neutral tone mark does not change but the height should be 10% of the height of the base character. In horizontal Zhuyin, the height of the space taken by the neutral tone mark does not change but the width should be 10% of the height of  the base character.</p>
-              <p data-lang="zh-hans" class="checkme">直排注音时，轻声符宽度不变，高度为基字的10%；横排注音时，轻声符的高度不变，宽度为基字的 10%。</p>
-              <p data-lang="zh-hant">直排注音時，輕聲符寬度不變，高度為基字的10%；橫排注音時，輕聲符的高度不變，寬度為基字的10%。</p>
+              <p data-lang="en">In vertical Zhuyin, the width of the space taken by the neutral tone mark does not change but the height to that of base characters ratio should be 1:15. In horizontal Zhuyin, the height of the space taken by the neutral tone mark does not change but the width to that of base characters ratio should be 1:15.</p>
```

Comment:
I suggest: "the height to that of base characters ratio should be " -> "the height ratio to the base character should be"

Reply:
@r12a Thanks, I made the changes.